### PR TITLE
Add initial deployment story.

### PR DESCRIPTION
**Pull request by bboe**

### CODE REVIEW

File: tasks/01_deployment_environment.md [hunk 1]
```diff
@@ -6,7 +6,62 @@ labels: in progress
...
+that server to have zero unscheduled downtime – all of the other steps work
+toward that effort.
+
+Please keep in mind that each feature delivers a modicum of value to either
+your users, or to aid in your development of the website. You want to deliver
+value to your users fast, so some features, like authentication, are saved
+until your users really need them.
+
+To begin with this story, please verify that things are set up for you by
+checking of each item below. If anything does not appear to work, it’s up to
```
Comment (rmacklin) [inline tasks/01_deployment_environment.md]: Typo: checking off

File: tasks/01_deployment_environment.md [hunk 2]
```diff
@@ -6,7 +6,62 @@ labels: in progress
...
+working on the story. You should be able to provide answers to them when the
+story is complete.
+
+The references might provide insight into the discussion topics, help you
+complete the story, or simply provide more information that you might be
+interested in. It is not expected that you read through all the references --
+you can if you want -- however, it might be a good idea to scan through them
+before getting started with a story.
+
+Happy hacking!
```
Comment (rmacklin) [inline tasks/01_deployment_environment.md]: "Hacking" can have a negative connotation in terms of the quality of the output. Consider "Happy coding!" instead

File: tasks/01_deployment_environment.md [hunk 3]
```diff
@@ -6,7 +6,62 @@ labels: in progress
...
+pipeline, there are three steps used for testing your application. Review
+applications are automatically created whenever you make a pull request. There
+can as many review applications as you have open pull requests. This step
+enables you to perform quality assurance (QA) tests while you are developing a
+feature. The second step, staging, is used to ensure no bugs are introduced as
+the result of a feature branch merge into master. The staging server is as
+close of a replica to your production server there is, so it provides a great
+way to ensure that changes will not negatively impact your customers. The
+staging server is automatically deployed whenever the master branch is updated
+and the automated tests on the updated master branch pass. The last step is
```
Comment (rmacklin) [inline tasks/01_deployment_environment.md]: Consider just "and the automated tests pass"

**Comment by bboe** [comment]
Thanks for the feedback @rmacklin! Comments addressed.

**Pull request merged**